orchestrator-salt/src/main/resources/salt/salt/resize/init.sls

```diff
@@ -0,0 +1,15 @@
+resize_volumes_script:
+  file.managed:
+    - name: /opt/salt/scripts/resize-storage-volumes.sh
+    - source: salt://resize/scripts/resize-storage-volumes.j2
+    - template: jinja
+    - makedirs: True
+    - mode: 755
+
+{% if "resize_disks" in grains.get('roles', []) %}
+
+execute_resize_volumes_initialize:
+  cmd.run:
+    - name: /opt/salt/scripts/resize-storage-volumes.sh
```

Comment:
Direct all output to a logfile aswell to get stderr output too, like here:

```
execute_mount_disks:
  cmd.run:
    - name: /opt/salt/scripts/mount-disks.sh 2>&1 | tee -a /var/log/mount-disks-allout.log && exit ${PIPESTATUS[0]}
```

Then add the logfile to the `vm-logs.json` so that the diagnostic tool can collect the logs

Reply:
done